doc/src/learn/index.md

```diff
@@ -16,7 +16,7 @@ For a deep dive into Sui technology, see the [Sui Smart Contracts Platform](http
 
 The following list includes the recent updates to Sui and the documentation:
 
-* [Bullshark](https://arxiv.org/abs/2201.05677) has replaced Tusk as the consensus component of the [Narwhal](https://github.com/MystenLabs/narwhal)-based [Sui consensus engine](../learn/architecture/consensus.md) for reduced latency and support for fairness with slower validators.
+* [Bullshark](https://arxiv.org/abs/2201.05677) replaced Tusk as the default consensus component of the [Narwhal](https://github.com/MystenLabs/narwhal)-based [Sui consensus engine](../learn/architecture/consensus.md) for reduced latency and support for fairness with slower validators. Note, Tusk may still be used.
```

Comment:
might still be used...
I'm interested in when and why it would be used if not the default.

Reply:
Another question for @huitseeker . Please explain why you think this should be might and the other reference can.

Reply:
> I'm interested in when and why it would be used if not the default.

Legitimate question: there are legitimate reasons to want to use Tusk, despite it not being the default. For example, it offers a more graceful performance degradation in the presence of Byzantine nodes. 

However, I think now is not the right timing to delve into "why you might want to use Tusk", because the current status of our implementation has some missing pieces (the current main hole being a random beacon) that Bullshark does not have.

Reply:
Thanks for confirming, @huitseeker